Controllers/JobOffer/Dtos/JobOfferDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace vabalas_api.Controllers.JobOffer.Dtos
+{
+    public class JobOfferDto
+    {
+        public int Id { get; set; }
+        public string CustomerName { get; set; } = string.Empty;
```

Comment:
We should remove string.Empty from our models or DTOs
If CustomerName param is not assigned then it will be "", and we break not null constriant in db.